Dockerfile

```diff
@@ -15,6 +15,6 @@ COPY torrc /etc/tor/
 HEALTHCHECK --timeout=10s --start-period=60s \
     CMD curl --fail --socks5-hostname localhost:9150 -I -L 'https://www.facebookcorewwwi.onion/' || exit 1
 
-EXPOSE 53 9150
+EXPOSE 53/udp 9150
```

Comment:
Sorry that if I didn't explain it clearly enough, as 9150 port is using tcp only, I wonder if we can have a pull request to directly specify the udp/tcp part, to ensure the consistency in the configs, thanks

Reply:
@PeterDaveHello 

Do you mean in Dockerfile
```
EXPOSE 53/udp 9150/tcp
```
and in docker-compose.yml
```
ports:
  -  127.0.0.1:53:53/udp
  -  127.0.0.1:9150:9150/udp
```

Right?

Reply:
Yes, I mean after this PR merging ;)

Reply:
(before the merging, I prefer to keep it without udp/tcp info to keep the consistency)